crates/astria-composer/src/config/searcher.rs

```diff
@@ -0,0 +1,119 @@
+use std::{
+    net::{
+        AddrParseError,
+        SocketAddr,
+    },
+    str::FromStr,
+};
+
+use figment::{
+    providers::{
+        Env,
+        Serialized,
+    },
+    Figment,
+};
+use serde::{
+    Deserialize,
+    Serialize,
+};
+use thiserror::Error;
+
+use super::cli;
+use crate::types::rollup::ChainId;
+
+const DEFAULT_SEQUENCER_URL: &str = "127.0.0.1:1317";
+const DEFAULT_API_URL: &str = "127.0.0.1:8080";
+const DEFAULT_CHAIN_ID: &str = "ethereum";
```

Comment:
should there be a default chain ID at all? i guess makes it easier for testing?

Reply:
yeah just easier for testing so you don't have to specify it as cli arg or env variable every time.